src/main/java/org/mdbenefits/app/utils/ExpenseCalculator.java

```diff
@@ -26,11 +26,12 @@ public BigDecimal totalUtilitiesExpenses() {
         Map<String, Object> inputData = submission.getInputData();
 
         expenses.forEach(val -> {
-            String inputFieldName = HomeExpensesType.getEnumByName(val).getInputFieldName();
-            expenseAmounts.add(
-                    new BigDecimal(inputData.getOrDefault(inputFieldName, "0").toString())
-                            .setScale(2, RoundingMode.HALF_UP));
-
+            if (!val.equalsIgnoreCase("None")) {
```

Comment:
@spokenbird I changed this to "equalsIgnoreCase()" so that we still do the right thing if someone accidentally changes this to "NONE" or "none".